README.md

```diff
@@ -3,6 +3,12 @@
 This is a Leiningen plugin to show the namespace dependencies of Clojure project
 sources as a graph.
 
+## Added Features
```

Comment:
Ah, this is because I'm merging their latest master into your branch, which has these additions to the README.

If you like I can just do a new PR against upstream?